packages/native/src/useLinking.tsx

```diff
@@ -70,6 +258,8 @@ export default function useLinking(
     };
   });
 
+  const [history] = React.useState(createMemoryHistory);
```

Comment:
You mean sth like `useMemoOne` here? Why not `x = useRef(); if x.current === null` etc.?

Reply:
Not really a memo since I only want initialize this once. Not using a ref because it's just extra code.